app/components/services/servicesController.js

```diff
@@ -17,6 +17,45 @@ function ($q, $scope, $state, Service, ServiceService, ServiceHelper, Notificati
     });
   };
 
+  $scope.forceUpdateAction = function(selectedItems) {
+    if ($scope.applicationState.endpoint.apiVersion <= 1.24) {
```

Comment:
As the force update action requires the API version to be 1.25+, we should hide the force update button if the API version does not match instead of displaying an error message. The button should be hidden in the services & service-details views.

Have a look at the sidebar for more details about hiding components when the API version does not match.

Reply:
I understand, the fact is that I don't know how to pass the apiVersion information to the servicesDatatable.html file as in the service-details view I already use an ngIf to show the button only if the version is >= 1.25. I'll have a look on how to pass that apiVersion info so I can use it in the datatable.

Reply:
You can add a binding into the `showForceUpdateButton` into the `servicesDatatable` component.

Then simply add `show-force-update-button=applicationState.endpoint.apiVersion >= 1.25` inside the datatable declaration.

Reply:
Ahhhh! Excelent I'll try thanks for the hint as always :D